### PR TITLE
Remove Azure.Core.props

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Review by christothes — APPROVED** [review]

**Review by jsquire — APPROVED** [review]

**Review by LarryOsterman — APPROVED** [review]

**Review by chamons — APPROVED** [review]

**Review by kinelski — APPROVED** [review]
Form Recognizer and Metrics Advisor LGTM.

File: sdk/mixedreality/Azure.MixedReality.Authentication/src/Azure.MixedReality.Authentication.csproj [hunk 1]
```diff
@@ -11,10 +11,15 @@
...
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" />
     <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+  </ItemGroup>
```
Comment (TimothyMothra) [inline sdk/mixedreality/Azure.MixedReality.Authentication/src/Azure.MixedReality.Authentication.csproj:21]: double package reference. this seems unintended?

**Review by maririos — APPROVED** [review]
TA and Translation Document LGTM

File: sdk/keyvault/Azure.Security.KeyVault.Administration/src/Azure.Security.KeyVault.Administration.csproj [hunk 2]
```diff
@@ -36,6 +37,4 @@
     <Compile Include="$(AzureCoreSharedSources)Base64Url.cs" LinkBase="Shared\Core" />
   </ItemGroup>
 
-  <!-- Import the Azure.Core project -->
-  <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Administration/src/Azure.Security.KeyVault.Administration.csproj:40]: Does this mean that any new features on which we need to depend need to be vetted and published before we can even use them? That seems overly prohibitive and will increase development time. Could we instead consider passing a variable from release runs that automatically swaps a ProjectRef to a PackageRef or something similar?

File: sdk/mixedreality/Azure.MixedReality.Authentication/src/Azure.MixedReality.Authentication.csproj (hunk 1 above)
Comment (pakrym) [reply sdk/mixedreality/Azure.MixedReality.Authentication/src/Azure.MixedReality.Authentication.csproj:21]: Yep, accident.

**Pull request merged**